lib/api/completeMultipartUpload.js

```diff
@@ -406,6 +407,11 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 multipart: true,
                 log,
             };
+            const replicationInfo = getReplicationInfo(objectKey, destBucket,
```

Comment:
since `getReplicationInfo` returns `undefined` if there is no replication config, you can avoid the conditional and set
```js
const metaStoreParams = {
     authInfo,
     objectKey,
     replicationInfo: getReplicationInfo(objectKey, destBucket, false, calculatedSize),
     ...
}
```